sdk/python/tests/compiler/testdata/katib.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
```

Comment:
Are we adding a custom DSL and YAML files? Is this something which already exists in KFP repo?

Reply:
This is based on this example since the kfp testdata didn't cover this edge case. https://github.com/kubeflow/pipelines/blob/master/components/kubeflow/katib-launcher/sample2.py

There's few more katib pipelines under samples, but they all require volumeOp as part of the pipeline.